h2d/Scene.hx

```diff
@@ -280,6 +289,11 @@ class Scene extends Layers implements h3d.IDrawable implements hxd.SceneEvents.I
 			}
 			if( !visible ) continue;
 
+			if (i.shape != null) {
+				pt.set((kx / max) * i.width + i.shapeX, (ky / max) * i.height + i.shapeY);

```

Comment:
maybe set pt after having the event.relX and event.relY are set, makes less code duplication.